syncopy/io/utils.py

```diff
@@ -119,22 +116,24 @@ def cleanup(older_than=24, interactive=True, **kwargs):
     for sk, sess in enumerate(sessions):
         sessid = allIds[sk]
         if sessid != __sessionid__:
-            with open(sess, "r") as fid:
-                sesslog = fid.read()
-            timestr = sesslog[sesslog.find("<") + 1:sesslog.find(">")]
-            timeobj = datetime.strptime(timestr, '%Y-%m-%d %H:%M:%S')
-            age = round((now - timeobj).total_seconds()/3600)   # age in hrs
-            if age >= older_than:
-                sesList.append(sess)
-                files = glob(os.path.join(__storage__, "*_{}_*".format(sessid)))
-                flsList.append(files)
-                ageList.append(round(age/24))                  # age in days
-                usrList.append(sesslog[:sesslog.find("@")])
-                ownList.append(sesslog[:sesslog.find(":")])
-                sizList.append(sum(os.path.getsize(file) if os.path.isfile(file) else
-                                   sum(os.path.getsize(os.path.join(dirpth, fname)) \
+            try:
+               with open(sess, "r") as fid:
+                    sesslog = fid.read()
+                    timestr = sesslog[sesslog.find("<") + 1:sesslog.find(">")]
+                    timeobj = datetime.strptime(timestr, '%Y-%m-%d %H:%M:%S')
+                    age = round((now - timeobj).total_seconds()/3600)   # age in hrs
+                    if age >= older_than:
+                        sesList.append(sess)
+                        files = glob(os.path.join(__storage__, "*_{}_*".format(sessid)))
+                        flsList.append(files)
+                        ageList.append(round(age/24))                  # age in days
+                        usrList.append(sesslog[:sesslog.find("@")])
+                        ownList.append(sesslog[:sesslog.find(":")])
+                        sizList.append(sum(os.path.getsize(file) if os.path.isfile(file) else sum(os.path.getsize(os.path.join(dirpth, fname)) \
                                        for dirpth, _, fnames in os.walk(file)
                                        for fname in fnames) for file in files))
+            except OSError as ex:
+                print(f"Unable to open {fid}: {ex}. (Maybe already deleted.)")
```

Comment:
Looking forward to proper logging.. but for now we could use `SPYInfo` but it's essentially just a print, so let's leave it like that